project1/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#Wordpress docker image with Php 7.4 and Apache
```

Comment:
Move this file to a folder that reflects what the content is related.

maybe `wordpress`...

Reply:
Done